mscore/scoreview.h

```diff
@@ -397,6 +396,8 @@ class ScoreView : public QWidget, public MuseScoreView {
       void changeState(ViewState);
 
       virtual const QRect geometry() const override { return QWidget::geometry(); }
+
+      void updateGrips();
```

Comment:
Why did you move it? Was it intentionally?

Reply:
Yes, I moved it so that it would be a public method rather than a private one.